doc/chapter-firststeps.asciidoc

```diff
@@ -21,6 +21,7 @@ usage: ./newsboat [-i <file>|-e] [-u <urlfile>] [-c <cachefile>] [-x <command> .
     -E, --export-to-file=<file>     export list of read articles to <file>
     -I, --import-from-file=<file>   import list of read articles from <file>
     -h, --help                      this help
+        --cleanup                   remove unreferenced items from cache
```

Comment:
Please change the Newsboat version in this listing to 2.22 (which is the next scheduled release). Otherwise someone might read it and think this was available since 2.10.

Reply:
Done: https://github.com/newsboat/newsboat/pull/1250/commits/7f30aa737b28ba48974d2f3a0e2c5b64e8e809b8